docs/config.yaml

```diff
@@ -70,8 +70,14 @@ streams:
       # building with a cosa too old to support `cosa remote-session`, which
       # should only be the case for RHCOS 4.11 and older.
       cosa_controller_img_hack: quay.io/coreos-assembler/coreos-assembler:latest
+      # OPTIONAL: override additional architectures to building for
+      additional_arches:
+        - s390x
     rawhide:
       type: mechanical
+      # OPTIONAL: additional architectures to skip building for
+      skip_additional_arches:
+        - s390x
```

Comment:
The use of the term `additional` in the toplevel key makes this key confusing. I would almost expect it to be `skip_arches` and not `skip_additional_arches`, but I understand tying it back to the name of the toplevel key.

I almost wonder if `secondary_arches` should be the toplevel key.

Reply:
Yeah, it's a bit awkward but I wanted to be consistent with the naming. It also conveys a bit more the fact that you can't skip x86_64, only arches that are valid in `additional_arches`.

Switching to `secondary` would indeed read a bit better I think. Let's do that in a follow-up?